modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/VerifyProductTask.java

```diff
@@ -29,65 +32,76 @@ public class VerifyProductTask extends DefaultTask {
 
 	@Input
 	@Optional
-	public String getBundleUrl() {
-		return _bundleUrl;
+	public WorkspaceExtension getExtension() {
+		return _extension;
 	}
 
 	@Input
 	@Optional
-	public String getDockerImageLiferay() {
-		return _dockerImageLiferay;
+	public String getProcut() {
+		return _product;
 	}
 
-	@Input
-	@Optional
-	public String getErrorMessage() {
-		return _errorMessage;
+	public void setExtension(WorkspaceExtension extension) {
+		_extension = extension;
 	}
 
-	@Input
-	@Optional
-	public String getTargetPlatformVersion() {
-		return _targetPlatformVersion;
+	public void setProduct(String product) {
+		_product = product;
 	}
 
-	public void setBundleUrl(String bundleUrl) {
-		_bundleUrl = bundleUrl;
-	}
+	@TaskAction
+	public void verifyProduct() throws Exception {
+		WorkspaceExtension.ProductInfo productInfo =
+			_extension.getProductInfo();
 
-	public void setDockerImageLiferay(String dockerImageLiferay) {
-		_dockerImageLiferay = dockerImageLiferay;
-	}
+		if (Objects.isNull(productInfo)) {
+			throw new GradleException(
+				"Can not get produtInfo for product '" + _product + "'");
```

Comment:
@simonjhy and @drewbrokke typo
`produtInfo` should be `productInfo`

Reply:
I will fix it.